easybuild/easyconfigs/h/HDF5/HDF5-1.8.12-foss-2018b.eb

```diff
@@ -0,0 +1,31 @@
+# Contribution from the Crick HPC team
+# uploaded by J. Sassmannshausen
+
+name = 'HDF5'
+version = '1.8.12'
+
+homepage = 'https://support.hdfgroup.org/HDF5/'
+description = """HDF5 is a data model, library, and file format for storing and managing data.
+ It supports an unlimited variety of datatypes, and is designed for flexible
+ and efficient I/O and for high volume and complex data."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'optarch': True, 'pic': True, 'usempi': True}
+
+source_urls = ['https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-%(version_major_minor)s/hdf5-%(version)s/src']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = [
+    'b5cccea850096962b5fd9e96f22c4f47d2379224bb41130d9bc038bb6c37dfcb',  # HDF5-tarball
+    '44e04cace87c54f75e8eb5b710679104f6703d88d10d6f3e5130b8cd90c69aa1',  # configure_libtool.patch
+]
+
+patches = [
+    'configure_libtool.patch',
+]
+
+dependencies = [
+    ('zlib', '1.2.8'),
```

Comment:
For 2018b toolchains use 1.2.11 instead, that is already available at GCCcore level.

Reply:
I thought of that but I was not sure if that might break things during the compilation of HDF5. I will give it a go as Travis has failed again.